views/movie.ejs

```diff
@@ -0,0 +1,24 @@
+<div class="jumbotron blackText">
+  <h1><%= movieResult.Title %></h1>
+  <p><strong>Year</strong> - <%= movieResult.Year %>, <strong>Rated</strong> <%= movieResult.Rated %><br>
+     <strong>Released</strong> - <%= movieResult.Released %><br>
+     <strong>Runtime</strong> - <%= movieResult.Runtime %><br><strong>Genre</strong> - <%= movieResult.Genre %><br>
+     <strong>Director</strong> - <%= movieResult.Director %> <br><strong>Writer</strong> - <%= movieResult.Writer %><br>
+     <strong>Actors</strong> - <%= movieResult.Actors %><br><strong>Plot</strong> - <%= movieResult.Plot %><br>
+     <strong>Awards</strong> - <%= movieResult.Awards %><br><strong></strong><br>
+     <strong>Metascore</strong> - <%= movieResult.Metascore %>
+     <input id="input-id" type="number" class="rating" min=0 max=5 step=0.5 data-size="lg" >
+
+     
+
+	 
+  <p><a class="btn btn-primary btn-lg" href="/show?search=<%=searchValue%>" role="button">Back</a></p>
+  <form action="/favorites" method="POST">
+    <input type="hidden" name="title" value="<%= movieResult.Title %>" />
+    <input type="hidden" name="imdbID" value="<%= movieResult.imdbID %>" />
+    <input type="hidden" name="year" value="<%= movieResult.Year %>" />
+    <p><a class="btn btn-primary btn-lg" role="button">Add to Favorites</a></p>
```

Comment:
Hi Scott, this was your problem right here. You'll need an input with type 'submit' to get the form to post, otherwise, nothing will happen.

```
    <input type="submit" class="btn btn-info" value="Add to Favorites">
```